uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -141,22 +141,33 @@ <h4 class="modal-title">New Document</h4>
                 </div>
               </div>
             </div>
+
+            <!-- Display only for manager role and hidden removed from DOM for other uasers -->
+            <div class="col-xs-6" ng-if="isRole('manager')">
```

Comment:
1. `hg-hide/ng-show`  make the element invisible, but still keep in DOM.
2. `ng-if` to remove the element from the DOM.